packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -196,6 +212,29 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
                 this.activateTerminal(termWidget);
             }
         }));
+
+        commands.registerCommand(TerminalCommands.TERMINAL_FIND_TEXT);
+        commands.registerHandler(TerminalCommands.TERMINAL_FIND_TEXT.id, {
+            isEnabled: () => this.shell.activeWidget instanceof TerminalWidgetImpl,
```

Comment:
please don't use implementations

I've filed https://github.com/theia-ide/theia/issues/5660 to get rid of `TerminalWidgetImpl` generally.